src/components/experiments/single-view/results/ActualExperimentResults.tsx

```diff
@@ -157,13 +157,15 @@ export default function ActualExperimentResults({
   const classes = useStyles()
   const theme = useTheme()
 
+  const availableAnalysisStrategies = [
+    AnalysisStrategy.IttPure,
+    AnalysisStrategy.MittNoCrossovers,
+    AnalysisStrategy.MittNoSpammers,
+    AnalysisStrategy.MittNoSpammersNoCrossovers,
+  ]
+  experiment.exposureEvents && availableAnalysisStrategies.push(AnalysisStrategy.PpNaive)
```

Comment:
I think this would be a bit more readable.

```suggestion
  if (experiment.exposureEvents) {
    availableAnalysisStrategies.push(AnalysisStrategy.PpNaive)
  }
```